### PR TITLE
Feature/bb bootstrap initial

### PR DESCRIPTION
**Pull request by bgbraga**
Initial version of bootstrap.
Steps automated by that version:

1) Force IPV4: today IPV6 is not a reality in most places. If you have both IPV4 and IPV6 enabled the connection sometimes go over IPV4 and sometime over IPV6. It could cause intermittent problems.

2) Extra DNS: Raspian O.S uses gateway (ex: 192.168.1.1) as default DNS. To avoid problems in networks that don’t have an external DNS configured in gateway, the bootstrap add two extra DNS: 8.8.8.8 and 8.8.4.4 that are provided by google and widely used.

3) Set a debian (raspian) locale just to avoid installation warnings

4) Update rasping O.S

5) Install NodeJS and NPM

6) Install Alsa

Please let’s test that before update the documentation.

Here I have problems with IPV6 (my gadgets only works with IPV4) and it could disable IPV6 and install everything needs to run TJBot. It is working fine.

### CODE REVIEW

File: bootstrap/bootstrap.sh [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+export LC_ALL="en_US.UTF-8"
+echo "en_US.UTF-8 UTF-8" | sudo tee -a /etc/locale.gen
+sudo locale-gen en_US.UTF-8
+
+#----update raspberry
+sudo apt-get update
+sudo apt-get -y dist-upgrade
+
+#----nodejs install
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```
Comment (victordibia) [inline bootstrap/bootstrap.sh]: I'll say we stick to setup_6.x ...
The last time i tried 7.x I had some installation errors. But 6.x works

Comment (victordibia) [inline bootstrap/bootstrap.sh]: Overall, I think this is a  good starting point for the bootstrap script. Thanks @bgbraga .
A few other things we still need to work on ...
- [Optional] Disable (blacklist) kernel sound module if we are using usb audio
- Manually install pigpio (software pwm library). For some reasons some pi OS updates dont come with this
- [Optional] Install opencv for local visual recognition

Comment (bgbraga) [inline bootstrap/bootstrap.sh]: yeah, I just see node 7.x is incompatible with node-gyp 
I will fix that.

Comment (bgbraga) [inline bootstrap/bootstrap.sh]: Ok ;)

I'm starting a new version with some improvements.

**Pull request merged**